addons/web_editor/static/src/js/editor/rte.summernote.js

```diff
@@ -4,6 +4,10 @@ odoo.define('web_editor.rte.summernote', function (require) {
 var Class = require('web.Class');
 const concurrency = require('web.concurrency');
 var core = require('web.core');
+// Use the top window's core.bus for dialog events so that they take the whole window
+if (window !== window.top) {
+    core = _.extend({}, core, {bus: window.top.odoo.__DEBUG__.services['web.core'].bus});
+}
```

Comment:
I would also explain in a comment that the events triggered on the bus in this file are meant to be listened by the bus of the top window (as you explained in your commit message).
In the end, I would leave `core` being the `core` of the iframe and I would define `const topBus = ...` here and use topBus in the file to be even more clearer.

Reply:
Done :+1: